PWGHF/TableProducer/candidateCreatorToXiPi.cxx

```diff
@@ -110,6 +110,10 @@ struct HfCandidateCreatorToXiPi {
     ccdb->setLocalObjectValidityChecking();
     lut = o2::base::MatLayerCylSet::rectifyPtrFromFile(ccdb->get<o2::base::MatLayerCylSet>(ccdbPathLut));
     runNumber = 0;
+
+    if (doprocessIdxCombinatorics == true && doprocessDerivedData == true) {
```

Comment:
```suggestion
    if (doprocessIdxCombinatorics && doprocessDerivedData) {
```

Reply:
One should also check that they are not both `false`.